src/Checkout.php

```diff
@@ -59,7 +59,7 @@ private function setPaymentUrl(array $data)
 			"order_id"       => $data['order_id'],
 			"redirect_url"   => $data['success_url'] ?? '',
 			"cancel_url"     => $data['cancel_url'] ?? '',
-			"webhooks"       => $data['webhooks'] ?? false,
+			"webhooks"       => $data['webhooks'] ?? 'false',
```

Comment:
@matiullah31 you should make this more robust. 
What if the developer passes in a Boolean instead of a string? we will have the same issue as before